src/main/java/org/jnode/fs/xfs/extent/DataExtent.java

```diff
@@ -34,20 +48,34 @@
      */
     private int state;
 
+    /**
+     * The msb.
+     */
+    private long msb;
```

Comment:
```suggestion
    private final boolean initialised;
```
The MSb of the extent determines if it has been initialised.

Reply:
The suggested change was applied,  thanks